src/notation/internal/notationparts.cpp

```diff
@@ -91,6 +91,10 @@ NotationParts::NotationParts(IGetScore* getScore, INotationInteractionPtr intera
     });
 }
 
+NotationParts::~NotationParts()
```

Comment:
why was it necessary?

Reply:
I added it for debugging purposes. A part is created and then immediately deleted...
I'll remove it.